src/moderation-plugin.tsx

```diff
@@ -196,8 +196,8 @@ export class ModerationPlugin extends KalturaPlayer.core.BasePlugin {
       this._pluginIcon = this.upperBarManager!.add({
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore
-        displayName: 'Moderation',
-        ariaLabel: 'Moderation',
+        displayName: 'Report Content',
+        ariaLabel: <Text id="moderation.report_content">Report Content</Text>,
```

Comment:
do we have this key in the i18n json files already ?

Reply:
Yes we have it here:
https://github.com/kaltura/playkit-js-moderation/blob/master/translations/en.i18n.json#L5